### PR TITLE
refactor: home blocks table

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Summary

https://app.clickup.com/t/861n1ttem

requires #495 to be merged

todo: 
- [x] stan
- [x] tests
- [x] disable loading state when polling after initial load

**Note: also adds polling for transactions**

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my UI changes in light AND dark mode
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

**Review by ItsANameToo — COMMENTED** [review]
- view all links to the transactions page and also to page 1 (should link to the next one really, e.g. page 2 onwards depending on the amount of entries per page the user chose)

**Comment by alexbarnsley** [comment]
good point :thinking: will sort it

File: resources/views/livewire/home/blocks.blade.php [hunk 1]
```diff
@@ -18,6 +18,7 @@ class="w-full"
         <x-tables.footers.view-all
             :results="$blocks"
             :count-suffix="trans('tables.home.blocks')"
+            :route="route('blocks', ['page' => 2])"
```
Comment (ItsANameToo) [inline resources/views/livewire/home/blocks.blade.php:21]: one risk with this if we don't calculate it, is that you could show 25 entries on the homepage while the subsequent pages have different amounts (10, 100) and then page 2 will not be what you expect it to be

Comment (ItsANameToo) [reply resources/views/livewire/home/blocks.blade.php:21]: ignoreeeeeee me

**Pull request merged**